treeherder/etl/text.py

```diff
@@ -10,7 +12,7 @@
 if six.PY3:
     filter_re = re.compile(r"([\U00010000-\U0010FFFF])")
 else:
-    filter_re = re.compile(ur"([\U00010000-\U0010FFFF])", re.U)  # noqa: E999
+    filter_re = re.compile(r"([\U00010000-\U0010FFFF])", re.U)
```

Comment:
It might be good to add a note similar to the commit message here that this is a unicode literal (effectively `ur` etc).  Thankfully the module isn't large but that future import implicitly makes this a unicode literal and might trip up future readers.

Reply:
Good idea :-)